django/db/models/fields/__init__.py

```diff
@@ -1812,14 +1812,27 @@ def _check_max_length_warning(self):
     def validators(self):
         # These validators can't be added at field initialization time since
         # they're based on values retrieved from `connection`.
+        validators_existed = super(IntegerField, self).validators
         range_validators = []
         internal_type = self.get_internal_type()
         min_value, max_value = connection.ops.integer_field_range(internal_type)
         if min_value is not None:
-            range_validators.append(validators.MinValueValidator(min_value))
+            min_value_existed = (min(
+                [validator.limit_value for validator in validators_existed
+                 if isinstance(validator, validators.MinValueValidator)] or
+                [float("-inf")]
+            ))
+            if min_value > min_value_existed:
+                range_validators.append(validators.MinValueValidator(min_value))
```

Comment:
Yes, it's better solution. But, as I understand, the second condition in the `if` statement should be `validator.limit_value >= min_value`?